.trunk/trunk.yaml

```
@@ -17,14 +17,14 @@ runtimes:
 # This is the section where you manage your linters. (https://docs.trunk.io/check/configuration)
 lint:
   enabled:
-    - checkov@3.2.424
-    - eslint@9.26.0
+    - checkov@3.2.427
+    - eslint@9.27.0
     - git-diff-check
-    - markdownlint@0.44.0
+    - markdownlint@0.45.0
     # - osv-scanner@2.0.2
```

```suggestion
    - osv-scanner@2.0.2
```
now that we removed the prepush hook we can finally reenable this